lib/index.ts

```diff
@@ -105,9 +105,22 @@ const nodeSdk = (...args: Parameters<typeof createSdk>) => {
      */
     async validateJwt(jwt: string): Promise<AuthenticationInfo> {
       // Do not hard-code the algo because library does not support `None` so all are valid
-      const res = await jwtVerify(jwt, sdk.getKey, { issuer: projectId, clockTolerance: 5 });
+      const res = await jwtVerify(jwt, sdk.getKey, { clockTolerance: 5 });
+      const token = res.payload;
 
-      return { jwt, token: res.payload };
+      if (token) {
+        token.iss = token.iss?.split('/').pop(); // support both url and project id as issuer
```

Comment:
Is this not something we are generating? 

In addition, can this URL end with `/`?
Maybe regex will be more accurate?

Reply:
- We do generate, but we should be backward compatibly in cases project id is provided and not url which contains the project id.
- It cannot end with `/` but I can add proper validation for such case

Reply:
👍